tests/e2e/cucumber/environment/index.ts

```diff
@@ -25,10 +25,18 @@ import {
   keycloakCreatedUser
 } from '../../support/store'
 import { Group, User } from '../../support/types'
-import { getTokenFromLogin, setAccessToken } from '../../support/utils/tokenHelper'
+import {
+  refreshAccessTokenForKeycloakOcisUser,
+  setAccessToken,
+  setAccessTokenForKeycloakOcisUser
+} from '../../support/utils/tokenHelper'
```

Comment:
import directly from `keycloak/ocisUserToken`

Reply:
I see that other code doesn't directly import  from API helper implementation. I guess this could be done.